spec/std/file_spec.cr

```diff
@@ -27,11 +27,35 @@ private def it_raises_on_null_byte(operation, &block)
   end
 end
 
+private def normalize_permissions(permissions, *, directory)
+  {% if flag?(:win32) %}
+    normalized_permissions = 0o444
+    normalized_permissions |= 0o222 if permissions.bits_set?(0o200)
+    normalized_permissions |= 0o111 if directory
+    File::Permissions.new(normalized_permissions)
+  {% else %}
+    File::Permissions.new(permissions)
+  {% end %}
+end
+
+private def to_windows_path(path)
```

Comment:
Maybe better `to_native_path`?

Reply:
This is also temporary, should be fixed up in whatever PR we add proper windows path support, hopefully #5550. So I don't think it matters.